ydb/src/client_topic/topicwriter/writer_options.rs

```diff
@@ -0,0 +1,57 @@
+// # Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a later release.
```

Comment:
You can remove the disclaimer, because rust sdk is non stable yet and any API can be changed without notice in new "minor" version.